shuffle/cli/src/main.rs

```diff
@@ -110,6 +118,16 @@ pub enum Subcommand {
     Build {
         #[structopt(short, long)]
         project_path: Option<PathBuf>,
+
+        #[structopt(short, long)]
+        network: Option<String>,
+
+        #[structopt(
+            short,
+            long,
+            help = "Network specific address to be used for publishing with Named Address Sender"
+        )]
+        address: Option<String>,
```

Comment:
`shuffle build` triggers codegen, which generates scripts and script functions that embed the sending address. To make it configurable, we allow one to pass in a desired address, if any.

Reply:
Just so i understand: the embedded sending address is the owner of the module that is being built?

Reply:
For `transaction-builer` aka codegen, the address that's embedded is the location of said script function as a result of deployment. ie: `"0xdeadbeef::Message::set_message"`. Owner == address under which module is deployed.

![image](https://user-images.githubusercontent.com/635121/143085564-9d23e7e3-a914-4d8b-b774-ccf0a1b0e219.png)